amplify/backend/backend-config.json

```diff
@@ -70,11 +70,6 @@
       "providerPlugin": "awscloudformation",
       "service": "Lambda"
     },
-    "createUserNotificationsData": {
-      "build": true,
-      "providerPlugin": "awscloudformation",
-      "service": "Lambda"
-    },
```

Comment:
We don't need the `createUserNotificationsData` lambda to be declared here?

Reply:
Whoops, thanks for catching this, I'll revert it!